### ISSUE TITLE
Cannot Connect to Server Using TeamTalk5Droid on Android 14

### ISSUE THREAD

**Issue by lbk2907** [BearWare/TeamTalk5#2067]
### Description

As stated in the issue title, I was not able to connect to server using TeamTalk5Droid on Pixel 7 Pro running Android 14. I can open TeamTalk, and change the setting. When I try to connect to any server, sometime the app will just crash, sometime it will like we enter to the server but the status is offline down there. Below is what I get in the server log.

> 2023-10-27 12:07:55.818639 User #3353 TCP address: ::ffff:175.141.97.223 connected.
2023-10-27 12:07:55.899036 User #3353 nickname: "LBK" username: "lbk2907" IP address: ::ffff:175.141.97.223 UDP address: ::ffff:175.141.97.223:46412 logged in.
2023-10-27 12:07:56.008422 User #3353 nickname: "LBK" username: "lbk2907" joined channel: "/".
2023-10-27 12:07:56.053819 User #3353 nickname: "LBK" username: "lbk2907" left channel: "/".
2023-10-27 12:07:56.053904 User #3353 nickname: "LBK" logged out.
2023-10-27 12:07:56.053928 User #3353 nickname: "LBK" disconnected.

I have try to clear all databut the issue still exists.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [X] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

It should able to connect to server.

### Actual behavior

Cannot connect to server using TeamTalk5Droid on Android 14.

### Steps to reproduce problem

Happend on TeamTalk V5.14.1 Build 125. TeamTalk DLL V5.14.1.5129

**Comment by CoBC** [comment]
It seams that the issue is only present in 5.14.1, not 5.14. I can't confirm because I haven't Android 14 on my phone yet, but another user has reported this issue only after update to 5.14.1.

**Comment by beqabeqa473** [comment]
I have android 14, on latest release version everything works as expected.

On 10/27/23, Corentin Bacqué-Cazenave ***@***.***> wrote:
> It seams that the issue is only present in 5.14.1, not 5.14. I can't confirm
> because I haven't Android 14 on my phone yet, but another user has reported
> this issue only after update to 5.14.1.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/issues/2067#issuecomment-1782887107
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by bear101** [comment]
Looks like a duplicate of #1986

**Comment by bear101** [comment]
Hm, not duplicate of #1986 since the server says that the client connects.
